recipe/doc/maintainer_guide.md

```diff
@@ -0,0 +1,15 @@
+# Guide for Maintainers of CUDA recipes
+
+This guide is intended for maintainers of the CUDA packages themselves.
+
+## Rationale for split packages
+
+In addition to the standardized dev/static division of libraries, some packages are also divided into multiple pieces for more specialized reasons.
+
+### nvcc split
+
+The `nvcc` compiler natively supports cross-compilation, i.e. a single host binary can produce binaries compiled for any target platform it supports without requiring a completely separate binutils installation for each target.
```

Comment:
Q: what does "single host binary" refer to here? This sentence is hard to parse...

Reply:
What I meant here is that the same nvcc binary can target different platforms, the way that clang can (but not gcc). I can simplify this sentence, it is probably only adding confusion.